### PR TITLE
Turf Icons Unit Test

### LINKED ISSUE #75372: Grass tiles no longer have a broken sprite (from burn dmg or explosions)
obraz
(broken grass tiles are supposed to look like sand or dirt turf - this was also the only way for players to achieve this look)
## Round ID:
[205485](https://scrubby.melonmesa.com/round/205485)

## Testmerges:
none

## Reproduction:
Damage a grass astroturf (easily done with a flamethrower).
The sprite does not change. You can now remove it via a crowbar instead of a shovel and it does not yield the tile.

### PR DESCRIPTION
**Pull request by ShizCalev**
unit tests turf icons to find... well, missing icons. honk.

fixes #75372
corrects a bunch of things messed up in #65504

:cl: ShizCalev
code: Made a new unit test to find turfs with broken/missing icons! Rejoice!
fix: Fixed a bunch of incorrect and missing turf icons.
/:cl:

### CODE REVIEW

File: code/game/turfs/open/floor/fancy_floor.dm [hunk 1]
```diff
@@ -198,7 +200,7 @@
 
 /turf/open/floor/fake_snow/Initialize(mapload)
 	. = ..()
-	AddElement(/datum/element/diggable, /obj/item/stack/tile/mineral/snow, 2, worm_chance = 0)
+	AddElement(/datum/element/diggable, /obj/item/stack/tile/mineral/snow, 2, worm_chance = 0) //TODO BEFORE MERGE: double check this is setting broken state properly.
```
Comment (ShizCalev) [inline code/game/turfs/open/floor/fancy_floor.dm:203]: just gotta double check these and we're all set.

File: code/game/turfs/open/snow.dm [hunk 2]
```diff
@@ -15,11 +16,10 @@
 
 /turf/open/misc/snow/Initialize(mapload)
 	. = ..()
-	AddElement(/datum/element/diggable, /obj/item/stack/sheet/mineral/snow, 2)
+	AddElement(/datum/element/diggable, /obj/item/stack/sheet/mineral/snow, 2) //TODO BEFORE MERGE: double check this is setting broken state properly.
```
Comment (ShizCalev) [inline code/game/turfs/open/snow.dm:19]: just gotta double check these and we're all set.

**Comment by ShizCalev** [comment]
Ready for review/merge.

**Comment by san7890** [comment]
kobe

**Pull request merged**